migration-from-4x.md

```diff
@@ -506,72 +575,63 @@ editor.windowManager.openUrl({
+| [`textcolor_cols`]({{site.url}}/docs-4x/plugins/textcolor/#textcolor_cols)                | [`color_cols`]({{site.baseurl}}/configure/content-appearance/#color_cols) |
+| [`textcolor_map`]({{site.url}}/docs-4x/plugins/textcolor/#textcolor_map)                  | [`color_map`]({{site.baseurl}}/configure/content-appearance/#color_map)   |
 
+### Changed plugins
 
-Please read the [docs]({{site.baseurl}}/plugins/) for more information on the Tiny 4.x configuration options.
-
-### Changed features
-
-These features have either changed or have been deleted in TinyMCE 5.0.
+These features have either changed or have been deleted in TinyMCE 5.
 
 | **Plugin name** | **Description** |
 | --------------- |  -------------- |
 | [ContextMenu]({{site.url}}/docs-4x/plugins/contextmenu/) | New API. See the [docs]({{site.baseurl}}/ui-components/contextmenu/). |
 | [ColorPicker]({{site.url}}/docs-4x/plugins/colorpicker/) | Moved to the core. See the [docs]({{site.baseurl}}/configure/content-appearance/#color_picker}}). |
 | [TextColor]({{site.url}}/docs-4x/plugins/textcolor/) | The `textcolor` plugin was removed and this setting is has been replaced by [`text_color`]({{site.baseurl}}/configure/content-appearance/#text_color) in TinyMCE 5.0. |
 
-#### Custom context toolbars
-
-The Context Toolbar accepts toolbar buttons that were previously added to the editor using the editor's `addButton`, `addToggleButton`, `addSplitButton` or a`ddMenuButton` functions. The method for creating custom context toolbars in TinyMCE 5.0 has also been changed from `editor.addContextToolbar()` to `editor.ui.registry.addContextToolbar()`.
-
-For information on Context Toolbars, see the [docs]({{site.baseurl}}/ui-components/contexttoolbar).
-
 #### Custom context menus
 
-The [Context menu]({{site.url}}/docs-4x/plugins/contextmenu/) is no longer a plugin, it is part of the core and always enabled. Where TinyMCE 4.x only supported adding registered menu items, the new context menu also allows plugins to register "sections" of the context menu. These sections are dynamic and may show or hide depending on the cursor position when the context menu is opened. For example, the default context menu config is now `'link image editimage table spelling'` which are all plugin section references, not menu items. For information on using context menus, see: [UI components - Context menu]({{site.baseurl}}/ui-components/contextmenu/).
+The [Context menus]({{site.url}}/docs-4x/plugins/contextmenu/) are part of the core and enable by default in TinyMCE 5. TinyMCE 5 supports adding registered menu items and allows plugins to register "sections" of the context menu. These sections show or hide depending on the cursor position when the context menu is opened. The default context menu consists of plugin section references, not menu items.
```

Comment:
`s/enable/enabled`?

Reply:
Also should this be `default context menu configuration`? As it sounds like it's saying the when the context menu is opened it'll show sections instead of menu items atm.

Reply:
It as a terrible sentence with little to no value that I tried to save.... and now it's gone.